banyand/stream/stream_query.go

```diff
@@ -36,11 +38,25 @@ var (
 
 type Query interface {
 	Stream(stream *commonv2.Metadata) (Stream, error)
+	StreamT(stream *commonv2.Metadata) (StreamT, error)
+}
+
+// StreamT is an interface exposed specifically for testing,
+// which also includes methods defined by the interface Stream
+//nolint:golint
+type StreamT interface {
```

Comment:
You don't need `StreamT` here. Pls, empower `Stream` instead. 

 * Move `io.Closer` into `Stream`
 * I will provide some paths to build write parameters for testing.

Reply:
After rebase #51 and #52 , this PR has been simpilified.

Pls have a review again.